tools/precompile/cldr_data.mjs

```diff
@@ -33,8 +33,8 @@ const addModifier = (keyCode, mod) => {
   return keyCode + modMap[mod];
 };
 
-const cldr2keycode = {
-  E00: 53,
+const cldr2keycode = (os) => ({
+  E00: (os == "osx") ? 100 : 53,
```

Comment:
Can you add some comments explaining this code? 

Also, I know it's not new here, but do we really not have constants for the keycodes?

Reply:
Yep, can add an explanation.

And nope, we do not have constants for keycodes. We have for a select few, but not for everything yet, no. Happy to have an issue about adding more constants. O:)